apps/src/netsim/NetSimShard.js

```diff
@@ -38,16 +38,25 @@ var NetSimShard = module.exports = function (shardID, pubSubConfig) {
   var channel = this.pubSub.subscribe(shardID);
 
   /** @type {NetSimTable} */
-  this.nodeTable = new NetSimTable(channel, shardID, 'n');
+  this.nodeTable = new NetSimTable(shardID, 'n', {
+    channel: channel
+  });
 
   /** @type {NetSimTable} */
```

Comment:
Please document these tables.  (Summarize what each of these tables is used for and give an example of a JSON value for each table.)